### PR TITLE
Int specs

### PR DESCRIPTION
**Pull request by squidgetx**
Pls review the code

### CODE REVIEW

File: src-cpp/client/client.cc [hunk 1]
```diff
@@ -7,13 +7,13 @@ static int pacallback(const void *inputBuffer, void* outputBuffer,
     PaStreamCallbackFlags statusFlags,
     void *userData) {
 
-  short *audio_out = (short*) outputBuffer;
-  std::deque<short> * play_buffer = (std::deque<short> *) userData;
+  int16_t *audio_out = (int16_t*) outputBuffer;
```
Comment (dhatch) [inline src-cpp/client/client.cc:10]: don't use c style casts 😢

File: src-cpp/client/client.cc [hunk 2]
```diff
@@ -36,7 +36,7 @@ static int pacallback(const void *inputBuffer, void* outputBuffer,
 }
 
 void Client::receiveFromFile() {
-  int num_read = file.read(file_buf, BUFFER_LENGTH);
+  sf_count_t num_read = file.read(file_buf, BUFFER_LENGTH);
```
Comment (dhatch) [inline src-cpp/client/client.cc:39]: 😄

File: src-cpp/master/main.cc [hunk 3]
```diff
@@ -16,7 +16,7 @@ int main(int argc, char *argv[])
   }
  
   string ip_addr = argv[1];
-  int port = atoi(argv[2]);
+  uint16_t port = atoi(argv[2]);
```
Comment (dhatch) [inline src-cpp/master/main.cc:19]: is `atoi` safe here?

File: src-cpp/master/master.cc [hunk 4]
```diff
@@ -12,16 +12,16 @@ using namespace std;
...
-  long num_read = file.read (buf, BUFFER_SIZE) ;
+  sf_count_t num_read = file.read (buf, BUFFER_SIZE) ;
 
   if (!num_read){
     return;
   }
 
 
-  long now = get_millisecond_time();
+  time_t now = get_millisecond_time();
```
Comment (dhatch) [inline src-cpp/master/master.cc:24]: 😭

File: src-cpp/util/mpacket.cc [hunk 5]
```diff
@@ -1,34 +1,34 @@
 #include "mpacket.h"
 
-MPacket::MPacket(long time, short* p, int ps) : timestamp(time), payload(p), payload_size(ps) {
+MPacket::MPacket(time_t time, int16_t* p, size_t ps) : timestamp(time), payload(p), payload_size(ps) {
 
 }
 
-int MPacket::get_payload_size() {
+size_t MPacket::get_payload_size() {
```
Comment (dhatch) [inline src-cpp/util/mpacket.cc:7]: `std::size_t`

File: src-cpp/master/master.cc (hunk 4 above)
Comment (charlierproctor) [inline src-cpp/master/master.cc:24]: to be fixed later with `mtime_t`

**Pull request closed without merging**